File: 3392-count-subarrays-of-length-three-with-a-condition/3392-count-subarrays-of-length-three-with-a-condition.py
class Solution:
    def countSubarrays(self, nums: List[int]) -> int:
        ans, n = 0, len(nums)

        for i in range(n - 2):
            if nums[i + 1] == (nums[i] + nums[i + 2]) * 2:
                ans += 1
        
        return ans

File: 3392-count-subarrays-of-length-three-with-a-condition/3392-count-subarrays-of-length-three-with-a-condition.cpp
class Solution {
public:
    int countSubarrays(vector<int>& nums) {
        int ans = 0, n = nums.size();

        for (int i = 0; i < n - 2; ++i){
            if (nums[i + 1] == (nums[i] + nums[i + 2]) * 2) ++ans;
        }

        return ans;
    }
};